Packages/com.unity.inputsystem/InputSystem/Plugins/XR/XRLayoutBuilder.cs

```diff
@@ -1,4 +1,4 @@
-#if UNITY_INPUT_SYSTEM_ENABLE_XR && ENABLE_VR
```

Comment:
Some places have `UNITY_FORCE_INPUTSYSTEM_XR_OFF` and some don't, is there a bigger picture where it's needed? or should it be in all places?

Reply:
think we've resolved that now.